<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Style for header area  ####################################################*/ 

QCalendarWidget QWidget {
	 alternate-background-color: #B8E2FF;
}

/* style for top navigation area ###############################################*/ 

#qt_calendar_navigationbar {
    background-color: #fff;
	border: 2px solid  #B8E2FF;
	border-bottom: 0px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
}

/* style for month change buttons ############################################ */

#qt_calendar_prevmonth, 
#qt_calendar_nextmonth {
	/* border delete */
    border: none;  
    /* delete default icons */
	qproperty-icon: none; 
	
    min-width: 13px;
    max-width: 13px;
    min-height: 13px;
    max-height: 13px;

    border-radius: 5px; 
	/* set background transparent */
    background-color: transparent; 
	padding: 5px;
}

/* style for pre month button ############################################ */

#qt_calendar_prevmonth {
	/* set text for button */
	/*qproperty-text: &amp;quot;&amp;gt;&amp;quot;;*/
	margin-left:5px;
	image: url(:/icon/arrow-119-48.ico);
}

/* style for next month button ########################################### */
#qt_calendar_nextmonth {
	margin-right:5px;
	image: url(:/icon/arrow-19-48.ico);
    /* qproperty-text: &amp;quot;&amp;gt;&amp;quot;; */
}
#qt_calendar_prevmonth:hover, 
#qt_calendar_nextmonth:hover {
    background-color: #55aaff;
}

#qt_calendar_prevmonth:pressed, 
#qt_calendar_nextmonth:pressed {
    background-color: rgba(235, 235, 235, 100);
}


/* Style for month and yeat buttons #################################### */

#qt_calendar_yearbutton {
    color: #000;
	margin:5px;
    border-radius: 5px;
	font-size: 13px;
	padding:0px 10px;
}

 #qt_calendar_monthbutton {
	width: 110px;
    color: #000;
	font-size: 13px;
	margin:5px 0px;
    border-radius: 5px;
	padding:0px 2px;
}

#qt_calendar_yearbutton:hover, 
#qt_calendar_monthbutton:hover {
    background-color: #55aaff;
}

#qt_calendar_yearbutton:pressed, 
#qt_calendar_monthbutton:pressed {
    background-color: rgba(235, 235, 235, 100);
}

/* Style for year input lineEdit ######################################*/

#qt_calendar_yearedit {
    min-width: 53px;
    color: #000;
    background: transparent;
	font-size: 13px;
}

/* Style for year change buttons ######################################*/

#qt_calendar_yearedit::up-button { 
	image: url(:/icon/arrow-151-48.ico);
    subcontrol-position: right;
}

#qt_calendar_yearedit::down-button { 
	image: url(:/icon/arrow-213-48.ico);
    subcontrol-position: left; 
}

#qt_calendar_yearedit::down-button, 
#qt_calendar_yearedit::up-button {
	width:10px;
	padding: 0px 5px;
	border-radius:3px;
}

#qt_calendar_yearedit::down-button:hover, 
#qt_calendar_yearedit::up-button:hover {
	background-color: #55aaff;
}

/* Style for month select menu ##################################### */

#calendarWidget QToolButton QMenu {
     background-color: white;

}
#calendarWidget QToolButton QMenu::item {
	/*padding: 10px;*/
}
 #calendarWidget QToolButton QMenu::item:selected:enabled {
    background-color: #55aaff;
}

#calendarWidget QToolButton::menu-indicator {
	/* Remove toolButton arrow */
      /*image: none; */
	nosubcontrol-origin: margin;
	subcontrol-position: right center;
	margin-top: 10px;
	width:20px;
}

/* Style for calendar table ########################################## */
#qt_calendar_calendarview {
	/* Remove the selected dashed box */
    outline: 0px;

	border: 2px solid  #B8E2FF;
	border-top: 0px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

#qt_calendar_calendarview::item:hover {
   border-radius:5px;
	background-color:#aaffff;
}

#qt_calendar_calendarview::item:selected {
    background-color: #55aa7f; 
	border-radius:5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>501</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: #e6f2fe ;  /* رنگ  */
    border: none;                             /* حذف حاشیه */
    border-radius: 10px;                      /* گرد کردن گوشه‌ها */
    color: black;                             /* رنگ متن داخل فیلد */
    padding: 6px 10px;                        /* فاصله داخلی */
    font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="setdate_pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Date</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
